src/app/common/components/AppBar/AppBar.tsx

```diff
@@ -1,32 +1,102 @@
+    text: string,
+    link: string,
+    newPage?: boolean
+};
 
 const AppBarComponent = () => {
-  const classes = useStyles();
+    const classes = useStyles();
+    const { history } = useUserFlow();
+    const stores = useStores();
+    const applicationStore: ApplicationStore = stores.Application;
+
+    const buttons: Button[] = [
+        { text: 'ssv.network', link: config.routes.HOME },
+        { text: 'join', link: config.routes.HOME },
+        { text: 'my account', link: config.routes.HOME },
+        { text: 'explorer', link: config.links.LINK_EXPLORER, newPage: true },
+        { text: 'docs', link: 'https://docs.ssv.network/', newPage: true },
+    ];
+    const menuButtons: Button[] = [
+        { text: 'Join the Network', link: config.routes.HOME },
+        { text: 'My Account', link: config.routes.HOME },
+        { text: 'Explorer', link: config.links.LINK_EXPLORER, newPage: true },
+        { text: 'Documentation', link: 'https://docs.ssv.network/', newPage: true },
+    ];
+    
+    const renderMenu = () => {
+        return (
+          <div className={classes.fullScreen}>
+            <Grid container spacing={0} justify="center" className={classes.menuDropDown}>
+              {menuButtons.map((button: Button, index: number) => {
+                      return (
+                        <Typography
+                          key={index}
+                          onClick={() => { switchPage(button.link, button.newPage); }}
+                          className={classes.menuButton}>
+                          {button.text}
+                        </Typography>
+                      );   
+            })}
+            </Grid>
+          </div>
+        );
+    };
+
+    const switchPage = (link: string, newPage: boolean = false) => {
+        if (newPage) {
+            window.open(link, '_blank');
+        } else {
+            history.push(link);
+        }
+        applicationStore.displayToolBarMenu(false);
+    };
+
+    const handleClick = (status: boolean) => {
+        applicationStore.displayToolBarMenu(status);
+    };
 
   return (
     <div className={classes.root}>
-      <AppBar position="static">
+      <AppBar className={classes.bloxColor} position="static">
         <Toolbar>
-          <Typography variant="h5" className={classes.title}>
-            <Logo />
-          </Typography>
-
-          <ConnectWalletButton />
-
-          <IconButton edge="end" className={classes.menuButton} color="inherit" aria-label="menu">
-            <MenuIcon />
-          </IconButton>
+          <Grid container direction="row" justify="space-around">
+            <Grid container className={classes.firstSection} justify="flex-start">
+              {buttons.map((button: Button, index: number) => {
+                        return (
+                          <Typography onClick={() => { switchPage(button.link, button.newPage); }} key={index} variant={index === 0 ? 'subtitle1' : 'subtitle1'} className={classes.button}>
```

Comment:
This line is too long.
In IDE you have vertical line which recommends to not go over it, usually it's 80 characters but check it.